src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURES_COUNTER = 5;
```

Comment:
Не очень информативное название константы
```suggestion
    public static final int NIMBER_OF_FIGURES = 5;
```